.github/workflows/publish.yml

```
@@ -8,7 +8,7 @@ jobs:
   test:
     strategy:
       matrix:
-        node: [12, 14]
+        node: [12, 15]
```

Please leave in the test matrix value for Node 14. Same goes for the matrices in `test.yml`.

```suggestion
        node: [12, 14, 15]
```